js/script.js

```diff
@@ -43,12 +50,15 @@ const handleTaskView = () => {
   $noTask.style.display = "none";
 
   toggleInputContainer(isVisible, handleAddTask);
-  if (!isVisible) renderTasks(tasks);
+  if (!isVisible)
+    renderTasks(
+      filteredOrSearchableTasks.length ? filteredOrSearchableTasks : tasks
```

Comment:
please make it a const and reuse it as it is being used in several places

Reply:
Done, bhai.